test/blackbox-tests/test-cases/directory-targets/copy-sandboxing.t

```diff
@@ -8,16 +8,11 @@ mode:
 
   $ cat >print-permissions.sh <<EOF
   > /usr/bin/env sh
-  > echo permissions of output/
-  > dune_cmd stat permissions output
-  > echo permissions of output/y
-  > dune_cmd stat permissions output/y
-  > echo permissions of output/x
-  > dune_cmd stat permissions output/x
-  > echo permissions of output/subdir
-  > dune_cmd stat permissions output/subdir
-  > echo permissions of output/subdir/z
-  > dune_cmd stat permissions output/subdir/z
+  > dune_cmd ls output
```

Comment:
I think this is equivalent to:

```
dune_cmd ls output/ output/subdir
```

Reply:
I've replaced it with an explicit `dune_cmd ls -d` so that we can see the perms on the dir too.